specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2016-05-01/Backups.json

```diff
@@ -0,0 +1,301 @@
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "BackupLocations.json#/parameters/BackupLocationParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/BackupList"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Backup.Admin/backupLocations/{location}/backups/{backup}": {
+            "get": {
+                "x-ms-examples": {
+                    "Returns a backup from a location based on name.": {
+                        "$ref": "./examples/Backups/Get.json"
+                    }
+                },
+                "description": "Returns a backup from a location based on name.",
+                "tags": [
+                    "Backups"
+                ],
+                "operationId": "Backups_Get",
+                "parameters": [
+                    {
+                        "$ref": "Backup.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "BackupLocations.json#/parameters/BackupLocationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/BackupParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "#/definitions/Backup"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Backup.Admin/backupLocations/{location}/backups/{backup}/restore": {
+            "post": {
+                "x-ms-examples": {
+                    "Get a backup from a location based on name.": {
```

Comment:
Please correct the description (it's restore backup not get backup, right?)

Reply:
Done.